biodatasets/osiris/osiris.py

```diff
@@ -0,0 +1,302 @@
+of the Creative Commons Attribution License
+Creative Commons Attribution 3.0 Unported License,
+which permits unrestricted use, distribution, and reproduction in any medium,
+provided the original work is properly cited (Furlong et al, BMC Bioinformatics 2008, 9:84).
+"""
+
+_HOMEPAGE = "https://github.com/rockt/SETH/tree/master/resources/OSIRIS"
+
+
+_LICENSE = "Creative Commons Attribution 3.0 Unported License"
+
+_URLS = {
+    _DATASETNAME: ["https://github.com/rockt/SETH/blob/master/resources/OSIRIS/corpus.xml?raw=true "
+                   ]}
+
+_SUPPORTED_TASKS = [Tasks.NAMED_ENTITY_RECOGNITION,
+                    Tasks.NAMED_ENTITY_DISAMBIGUATION]
+
+
+_SOURCE_VERSION = "1.2.0"
+
+_BIGBIO_VERSION = "1.0.0"
+
+
+class Osiris(datasets.GeneratorBasedBuilder):
+    """
+    The OSIRIS corpus is a set of MEDLINE abstracts manually annotated
+    with human variation mentions.
+    """
+
+    SOURCE_VERSION = datasets.Version(_SOURCE_VERSION)
+    BIGBIO_VERSION = datasets.Version(_BIGBIO_VERSION)
+
+    # You will be able to load the "source" or "bigbio" configurations with
+    # ds_source = datasets.load_dataset('my_dataset', name='source')
+    # ds_bigbio = datasets.load_dataset('my_dataset', name='bigbio')
+
+    # For local datasets you can make use of the `data_dir` and `data_files` kwargs
+    # https://huggingface.co/docs/datasets/add_dataset.html#downloading-data-files-and-organizing-splits
+    # ds_source = datasets.load_dataset('my_dataset', name='source', data_dir="/path/to/data/files")
+    # ds_bigbio = datasets.load_dataset('my_dataset', name='bigbio', data_dir="/path/to/data/files")
+
+    BUILDER_CONFIGS = [
+        BigBioConfig(
+            name="osiris_source",
+            version=SOURCE_VERSION,
+            description="osiris source schema",
+            schema="source",
+            subset_id="osiris",
+        ),
+        BigBioConfig(
+            name="osiris_bigbio_kb",
+            version=BIGBIO_VERSION,
+            description="osiris BigBio schema",
+            schema="bigbio_kb",
+            subset_id="osiris",
+        )
+    ]
+
+    DEFAULT_CONFIG_NAME = "osiris_source"
+
+    def _info(self) -> datasets.DatasetInfo:
+
+        if self.config.schema == "source":
+
+            features = datasets.Features(
+                {
+                    "Pmid": datasets.Value("string"),
+                    "Title": datasets.Value("string"),
+                    "Abstract": datasets.Value("string"),
+                    "genes": [
+                        {
+                            "g_id": datasets.Value("string"),
+                            "g_lex": datasets.Value("string"),
+                            "offsets": [[datasets.Value("int32")]]
+                        }
+                    ],
+                    "variants": [
+                        {
+                            "v_id": datasets.Value("string"),
+                            "v_lex": datasets.Value("string"),
```

Comment:
The "v_norm" field is missing. Variants are in the form:  
```html
"< variant v_id="660339" v_lex="Ala55Val" v_norm="A55V" > Ala55Val </ variant >"
```

Reply:
Thank you! I have forgotten about v_norm but I now everything looks fine.